guides/v2.3/magento-functional-testing-framework/release-2/commands/mftf.md

```diff
@@ -104,6 +104,7 @@ opt                | description|
 `--force`           | force generation of tests regardless of Magento Instance Configuration  
 `--lines=LINES`     | Used in combination with a parallel configuration, determines desired group size [default: 500]  
 `--tests=TESTS`     | A parameter accepting a JSON string used to determine the test configuration  
+`--debug`           | Generates tests with extra schema validation to narrow down issues
```

Comment:
@aljcalandra 
The description is not clear. Please rephrase. Use more than one sentence if needed.